### PR TITLE
Yesterday, I died after I traded my antag for morph, so now I grudgecoded morphs to be more powerful

### PR DESCRIPTION
**Pull request by Arzion5**
:cl: Salty Boi
tweak: Morph has been severely buffed. It deals more damage, moves faster in its original form, has more health, destroys objects faster, eats corpses and items faster, has a shorter cooldown on transformations, and eating corpses now heals it for half its health
/:cl:

Beg me for mercy

### CODE REVIEW

**Comment by Spankmaster1066** [comment]
I dono they're fairly OP as is.

**Comment by ghost** [comment]
I agre with spanking,  they're cancer fast,  make them slow but stronger tbh.  Slow,  even while transformed, but strong enough to not be killed as soon as it tries to climb a vent.  Also make them unpushable so they can climb vents in peace.

**Comment by JohnGinnane** [comment]
Original code: https://github.com/HippieStation/HippieStation/blob/master/code/game/gamemodes/miniantags/morph/morph.dm

File: hippiestation/code/game/gamemodes/miniantags/morph/morph.dm [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+mob/living/simple_animal/hostile/morph
+	speed = 1
+	maxHealth = 250
+	health = 250
+	obj_damage = 100
+	melee_damage_lower = 35
+	melee_damage_upper = 35
+	armour_penetration = 100 //Pain
+
+/mob/living/simple_animal/hostile/morph/AttackingTarget()
```
Comment (JohnGinnane) [inline hippiestation/code/game/gamemodes/miniantags/morph/morph.dm:12]: Current code: https://github.com/HippieStation/HippieStation/blob/master/code/game/gamemodes/miniantags/morph/morph.dm#L186

This is quite a bit more powerful, much faster attacking

**Comment by JohnMcLaime** [comment]
Make them sloooooow, they can ventcrawl anyway, so that should be their main means of escaping, this will also promote with using their mimic ability for actual blending in and stealth attacks. Dont give them speed, it nulifies their speciality.

**Comment by ghost** [comment]
nah

**Comment by KayeArray** [comment]
nah

**Pull request closed without merging**